crates/astria-core/src/protocol/transaction/v1/action/mod.rs

```diff
@@ -1571,6 +1555,15 @@ impl Protobuf for BridgeLock {
             .fee_asset
             .parse()
             .map_err(BridgeLockError::invalid_fee_asset)?;
+        if proto.destination_chain_address.is_empty() {
+            return Err(BridgeLockError::field_not_set(
+                "destination_chain_address",
+            ));
+        }
+        if proto.destination_chain_address.as_bytes().len() != 256 {
```

Comment:
I think 256 bytes is a reasonable upper limit, but we shouldn't restrict to an exact meassure.